community/modules/compute/gke-node-pool/variables.tf

```diff
@@ -80,6 +80,20 @@ variable "image_type" {
   default     = "COS_CONTAINERD"
 }
 
+variable "local_ssd_count" {
```

Comment:
I would suggest the name `ephemeral_storage_local_ssd_count` to more closely tie it to the terraform API naming (`ephemeral_storage_local_ssd_config`) and to make the distinction clear between this and `local_nvme_ssd_block_config`.

Reply:
Yes, that occurred to me too, as I was reviewing the final patch, but I decided to wait and see if the patch had favorable response before nitpicking on the naming.  I'll make the change.